### ISSUE TITLE
"-a" command line option isn't implemented

### ISSUE THREAD

**Issue by Johni0702** [ReplayMod/ReplayStudio#6]

**Comment by T-x-T** [comment]
Are there any plans to implement this? Or is there some other thing I can do to concatenate multiple replays?
This would make it much easier to make timelapses of huge builds that cant be built in a single session

**Comment by Johni0702** [comment]
No, there are no plans to implement this at the moment (it's not just the command line option which isn't implemented, it's that there's no code whatsoever to concatenate replays). It would probably not be particularly difficult to implement though.

**Comment by T-x-T** [comment]
Great, that seems like semi-good news then :P 
I would be happy to sponsor the feature financially (30-60$), if its going to happen

**Comment by T-x-T** [comment]
Just a FYI: I forked the repo to work on the append option myself. So far I have managed to get a basic version going. I "just" need to fix a few problems, but I think I can open a PR in the coming days

**Comment by nerdfoundrygaming** [comment]
LOL after a day of messing with the tool I opted to look at the issues.... Wish I looked sooner!

Hopefully @T-x-T has something soon, desperately need this feature!

**Comment by T-x-T** [comment]
Well... Sorry to disappoint, I never got around to fix the "few" problems that my implementation has. If you have some java knowledge you can probably clone my fork, compile it and then try it out. (Please back up your replays before that)
IIRC players from each separate replay get retained, resulting in duplicate players (this gets fixed each time the dimension changes, _I think_)
I might need this in the future, I will make sure to @ you whenever I get around to actually implementing that feature @nerdfoundrygaming

**Comment by McModknower** [comment]
As long as this isn't implemented, it would be nice if this is mentioned in the help text of that command line option.
